include/onnxruntime/core/providers/vitisai/vitisai_provider_factory.h

```diff
@@ -10,7 +10,9 @@ extern "C" {
 /**
  * \param use_arena zero: false. non-zero: true.
  */
-ORT_API_STATUS(OrtSessionOptionsAppendExecutionProvider_VITISAI, _In_ OrtSessionOptions* options, const char *backend_type, int device_id);
+ORT_API_STATUS(OrtSessionOptionsAppendExecutionProvider_VITISAI, _In_ OrtSessionOptions* options,
+               const char* backend_type, int device_id, const char* export_runtime_module,
+               const char* load_runtime_module);
 
```

Comment:
Might be worth putting the options in a struct as that's what we're doing for a number of other EPs now.

[https://github.com/microsoft/onnxruntime/blob/b854f2399dafb379584a0d7aebe42c958ae5aede/include/onnxruntime/core/session/onnxruntime_c_api.h#L269](https://github.com/microsoft/onnxruntime/blob/b854f2399dafb379584a0d7aebe42c958ae5aede/include/onnxruntime/core/session/onnxruntime_c_api.h#L269)

Reply:
Ok, you want me to do that in this PR still?

Reply:
Whenever - not required in this PR. Just something that may grow better over time when you want to switch to that.

Reply:
Ok, it looks like all tests have passed so maybe we merge the PR in like this and I will put all options in a struct in a future PR?